lerna.json

```diff
@@ -3,5 +3,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "45.1.0"
```

Comment:
It looks like lerna has been run - not sure if it's a problem though? I guess it will be overridden on release?